department-of-veterans-affairs/caseflow-feedback
# Add original_url as feedback attribute and write it in github issue

Connected to https://github.com/department-of-veterans-affairs/caseflow/issues/8893

Currently, build is failing because of this:
https://github.com/department-of-veterans-affairs/caseflow-feedback/pull/196

To test:
Go to http://localhost:3000/?redirect=http%3A%2F%2Fwww.caseflowdemo.com%2Fhelp&subject=Caseflow&original_url=https%3A%2F%2Fappeals.cf.ds.va.gov%2Fqueue%2Fappeals%2F81cee687-820f-430f-8607-197dc9c973db, 
leave feedback and submit it. 
This action should create a record in local database with original_url: https://appeals.cf.ds.va.gov/queue/appeals/81cee687-820f-430f-8607-197dc9c973db
